examples/jdbc-client-crud-operations/jdbc_client_crud_operations.bal

```diff
@@ -110,6 +109,23 @@ public function main() {
         io:println("Select data from student table failed: ",
                 <string> err.detail()["message"]);
     }
+
+    // The query parameters are given as arguments of the type `jdbc:Parameter`
+    // for the `select` remote function.
+    // Default direction is IN.
+    jdbc:Parameter p3 = { sqlType: jdbc:TYPE_INTEGER, value: 35 };
```

Comment:
Shall we reformat this? We should not have space after `{`. May be we need to reformat the entire file